loculus_values/environment_specific_values/production.yaml

```diff
@@ -69,10 +69,10 @@ podPriorityClassName: production
 resources:
   website:
     requests:
-      memory: "200Mi"
+      memory: "1Gi"
       cpu: "1000m"
     limits:
-      memory: "1Gi"
+      memory: "5Gi"
```

Comment:
This may be necessary ATM due to non-ideal coding. I think we should aim to limit at less than this in the future (going more for having many replicas). [You can resolve this comment immediately, not requesting changes]

Reply:
Indeed, only necessary due to lack of pagination - but given we require that much RAM sometimes we should also grant it.